chainer/testing/helper.py

```diff
@@ -76,6 +76,24 @@ def without_requires(*requirements):
     return unittest.skipIf(skip, msg)
 
 
+def is_requires_satisfied(*requirements):
```

Comment:
How about implement like [this](https://github.com/cupy/cupy/blob/f922131a20f7edfd456c08c3c84ce1ca8137ae62/cupy/testing/helper.py#L960)?

Reply:
To use from `:skipif:`, the function has to return a boolean, instead of applying to a `TestCase`.